_layouts/pages/docs.html

```diff
@@ -1,8 +1,7 @@
 ---
 layout: page
 scripts:
-  - https://cdn.jsdelivr.net/docsearch.js/1/docsearch.min.js
-  - /js/docsearch.js
+  - "/js/build/documentation.js"
```

Comment:
I wanted to do that, but didn’t figure out how it’s done yet, your proposal sounds good.